### PR TITLE
[17.0][MIG] event_registration_cancel_reason: Migration to 17.0

### PR DESCRIPTION
**Pull request by carolinafernandez-tecnativa**
Standard migration to v17

@Tecnativa
TT49911
@victoralmau @pilarvargas-tecnativa

### CODE REVIEW

File: event_registration_cancel_reason/views/event_registration_view.xml [hunk 1]
```diff
@@ -10,8 +10,8 @@
                     <field
                         name="cancel_reason_id"
                         class="oe_inline"
-                        options="{'no_open':True'}"
-                        attrs="{'invisible':[('cancel_reason_id','=',False)]}"
+                        options="{'no_open': True}"
+                        invisible="cancel_reason_id == False"
```
Comment (chienandalu) [inline event_registration_cancel_reason/views/event_registration_view.xml]: Careful, it's not the same
```suggestion
                        invisible="not cancel_reason_id"
```

**Review by ivantodorovich — APPROVED** [review]
LG

**Review by chienandalu — APPROVED** [review]
Tested :+1:

**Pull request merged**